[FIX]: Add missing .order() and .single() mocks in API tests

### Bug Description

Two API tests are currently failing due to missing Supabase query mocks. The test suite runs at 49/50 instead of 50/50.

Failing tests:
- certificates.test.ts — missing .order() mock
- meters.test.ts > POST /api/meters > crea medidor válido → 201 — missing .single() mock

### Steps to Reproduce

1. Clone the repo and install dependencies
2. cd apps/web
3. Run pnpm run test:api
4. Observe 1 failing test in meters.test.ts

### Expected Behavior

All 50 tests pass. pnpm run test:api exits with code 0.

### Actual Behavior

49/50 tests pass. meters.test.ts fails with:
AssertionError: expected 500 to be 201
[Unhandled Error] supabase.from(...).select(...).eq(...).single is not a function

### Component

Other

### Severity

Low (minor issue)

### Environment

The mock chain needs .single() added for the meters POST route and .order() for the certificates query. Follow the existing Vitest + Supabase mock patterns already used in the test files.

### Error Logs

```shell

```

### Additional Context

_No response_

--- a/apps/web/__tests__/api/certificates.test.ts
+++ b/apps/web/__tests__/api/certificates.test.ts
@@ -7,7 +7,7 @@ const COOP_ID = "00000000-0000-0000-0000-000000000001"
 const { mockSingle, mockOrder, mockFrom, mockLimit } = vi.hoisted(() => {
   const mockSingle = vi.fn()
   const mockLimit = vi.fn(() => ({ data: [] as unknown[], error: null }))
-  const mockOrder = vi.fn(() => ({ data: [], error: null }))
+  const mockOrder = vi.fn(() => ({ data: [], error: null, single: mockSingle, order: mockOrder, eq: mockEq, gte: mockEq, lte: mockEq, limit: mockLimit }))
   const mockEq: ReturnType<typeof vi.fn> = vi.fn(() => ({
     order: mockOrder,
     eq: mockEq,

--- a/apps/web/__tests__/api/meters.test.ts
+++ b/apps/web/__tests__/api/meters.test.ts
@@ -8,7 +8,7 @@ const { mockSingle, mockOrder, mockIn, mockFrom } = vi.hoisted(() => {
   const mockSingle = vi.fn()
   const mockIn = vi.fn(() => ({ data: [], error: null }))
   const mockOrder = vi.fn(() => ({ data: [], error: null, in: mockIn }))
-  const mockEq: ReturnType<typeof vi.fn> = vi.fn(() => ({ order: mockOrder, eq: mockEq, in: mockIn }))
+  const mockEq: ReturnType<typeof vi.fn> = vi.fn(() => ({ single: mockSingle, order: mockOrder, eq: mockEq, in: mockIn }))
   const mockSelect = vi.fn(() => ({ single: mockSingle }))
   const mockInsert = vi.fn(() => ({ select: mockSelect }))
   const mockFrom = vi.fn(() => ({
@@ -107,6 +107,7 @@ describe("POST /api/meters", () => {
       technology: "solar",
       capacity_kw: 5,
     }
+    mockSingle.mockResolvedValueOnce({ data: { id: "prosumer-id" }, error: null })
     mockSingle.mockResolvedValueOnce({ data: fakeMeter, error: null })
 
     const res = await POST(

